Header header
string reason
int32  id
string magnet